### ISSUE TITLE
Accounting

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#132]
Working out the best methods to manage the accounts.

**Comment by markwilliamfirth** [comment]
I noticed Xero are currently doing a Black Friday deal until midnight tonight which prompted me to post this:

https://www.xero.com/uk/pricing/
"Black Friday sale: Get 75% off any price plan plus free payroll, for three months.*
Offer ends midnight, Wednesday 30 November. Use promo code BF2016
Cancel or change your plan at any time."

Although I understand (From Ines) there may be difficulties involved with extracting and inputing data into tools like FreeAgent and Xero, especially due to the need for integration of contribution-tracking apps.

Posting this here though as an open issue as it's likely possible to improve upon the current manual process in some way and considering alternative tools might be something to do in future after recognising the requirements needed and limitations of such tools.

**Comment by nelsonic** [comment]
@markwilliamfirth that sounds like a _great_ idea as a starting point for the accounts. 
@iteles uses Xero for F&C and others also have _experience_ with it so we should make a _decision_ today!

**Comment by nelsonic** [comment]
@iteles we discussed this _verbally_, but it's worth _summarising_ in the issue to ensure everyone is _aware_ of our thought-process ... 
_One_ of the _reasons_ we don't _already_ have an **Accounting System** is that we (_foolishly_) want to _find_ ***OR*** _**build**_ our own **open-source** tool to keep track our Income/Expenditure. 
But this noble yet _naive_ goal is holding us back from just getting the _task_ done. 
I _agree_ with @markwilliamfirth we need to _focus_ on our "_core_" product and use an _existing_ SAAS for _everything_ else. 
So, in conclusion, we `should` sign up to `Xero` (_the "logical" choice_) and use it for at least 

`Xero` has an **API**: https://developer.xero.com/documentation/api/api-overview/
So we _can_ extract our data in the future when we _gradually_ build our own tools.
But for now, in the interest of recovering Inês' _sanity_ this is a _good_ decision.

> relates to: https://github.com/dwyl/hq/issues/92

**Comment by markwilliamfirth** [comment]
Let's test out using Xero and reopen if it doesn't work out!

**Comment by nelsonic** [comment]
Sadly Xero was adopted in order to "put out a fire" and no real _system_ was created.
Rather than having a clockwork management accounting system that _effortlessly_ produces the statutory financial accounts, we have a _mess_ that has incorrect data and unreliable reports. 
I blame myself for _allowing_ this to happen.
